### DISCUSSION TITLE
postal_smtp_1 Exit 1

### DISCUSSION THREAD

**Discussion by DanteAnuj** [postalserver/postal#1832] [Help with using Postal] [answered]
i was trying installing postal then I encountered this error when I run the **postal status** command

error -

_postal_smtp_1 /docker-entrypoint.sh post Exit 1_

![postalissue](https://user-images.githubusercontent.com/58907150/151153615-9c1a866c-4d5c-4249-98dd-2b56dd302159.png)

Screenshot link (in case above doesn't work) - https://prnt.sc/26j9ue9

I tried - **postal smtp logs** 

Screenshot - https://prnt.sc/26jkd15

Logs Here
https://pastebin.com/ctxS1HhE

**Comment by mattbaylor** [comment, marked as answer]
Add the following to the smtp_server section of your postal.yml config file:

```
bind_address: '0.0.0.0'
```

My guess is you're running into an IPV6 binding issue.

**Comment by DanteAnuj** [comment]
Thanks, Matt i  binded my VPS IP 
restarted postal it worked !!!

**Comment by willpower232** [comment]
it looks like something may already be using port 25 so you should double check that only Postal is using it

**Reply by DanteAnuj** [reply]
> it looks like something may already be using port 25 so you should double check that only Postal is using it

What the command for it please and thank you

**Reply by willpower232** [reply]
`sudo netstat -lnp` should tell you everything listening on a port on your server.
